DOCUMENTATION.md

```diff
@@ -181,10 +181,6 @@ If the request is not successful, the client returns a `Notify.Exceptions.Notify
 
 ### Method
 
-```csharp
-EmailNotificationResponse response = client.SendEmail(emailAddress, templateId, personalisation, reference, emailReplyToId);
```

Comment:
If we remove this example, maybe the line below now needs to start with `EmailNotificationResponse response = client.SendEmail ...` to keep things consistent with send letter and send sms